src/components/Navbar.js

```diff
@@ -79,6 +80,20 @@ const Navbar = class extends React.Component {
                 Contact
               </Link>
             </div>
+            <div className="navbar-end is-paddingless" style={{paddingTop: '20px', marginLeft: '33%'}}>
```

Comment:
In bulma inside of a `navbar-menu` there is supposed to be only one `navbar-start` and one `navbar-end`.

See documentation:
https://bulma.io/documentation/components/navbar/